docs/docs-components/contexts/DocsExperimentProvider.js

```diff
@@ -8,7 +8,7 @@ import { ExperimentProvider } from 'gestalt';
  * - Unless you want the experimental behavior live on the docs for everyone, REMOVE YOUR EXPERIMENT HERE before merging your PR!
  * */
 
-const enabledExperiments = [];
+const enabledExperiments = ['web_gestalt_redesigned_badge', 'mweb_gestalt_redesigned_badge'];
```

Comment:
See the note above — we should remove these so that users aren't confused why they're seeing different UI

Reply:
This will be replaced with logic from https://github.com/pinterest/gestalt/pull/2511